sdk/storage/Azure.Storage.Files.DataLake/api/Azure.Storage.Files.DataLake.netstandard2.0.cs

```diff
@@ -604,6 +619,13 @@ public partial class FileSystemProperties
         public Azure.Storage.Files.DataLake.Models.DataLakeLeaseStatus? LeaseStatus { get { throw null; } }
         public System.Collections.Generic.IDictionary<string, string> Metadata { get { throw null; } }
         public Azure.Storage.Files.DataLake.Models.PublicAccessType? PublicAccess { get { throw null; } }
+        public int? RemainingRetentionDays { get { throw null; } }
+    }
+    [System.FlagsAttribute]
+    public enum FileSystemStates
+    {
+        None = 0,
+        Deleted = 1,
```

Comment:
does this work like "include" in blobs or "show only" like in other HNS soft delete?

Reply:
include.